### PR TITLE
Added exercise 'math-exerciser'

### PR DESCRIPTION
**Pull request by ahnlabb**
- Basic readme with outline of the exercise
- Basic solution file
- Example showing what a more advanced solution might look like

### CODE REVIEW

File: exercises/math-exerciser/solution.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from random import randint
+from operator import add
+
+min_number = 0
+max_number = 10
+questions = [('Vad är "{} + {}"?', add)]
+
+selected_question = questions[randint(0, len(questions) - 1)]
```
Comment (ErikBjare) [inline exercises/math-exerciser/solution.py]: Extract the question index into a variable to make it clearer.

File: exercises/math-exerciser/solution.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+number1 = randint(min_number, max_number)
+number2 = randint(min_number, max_number)
+
+user_answer = input(selected_question[0].format(number1, number2) + ": ")
+actual_answer = selected_question[1](number1, number2)
```
Comment (ErikBjare) [inline exercises/math-exerciser/solution.py]: Probably unclear what happens here for someone new to programming.

**Comment by ahnlabb** [comment]
@ErikBjare Agreed, the simplest solution should be trivial, but it also needs to be very clean. My vision is an almost neverending exercise with lots of small improvements that the students can make as they learn more if the idea appeals to them. Having a few such exercises

**Comment by gorelhedin** [comment]
Looks good! @ErikBjare, will you merge it?

**Comment by ErikBjare** [comment]
I'll merge it if @ahnlabb thinks that it's ready. Do you want to add the few fixes we talked about?

 - [ ] Pre-generate question & answer tuples in a function (gets rid of dynamically calling the `add` function)
 - [ ] Extract question index into variable

**Comment by ahnlabb** [comment]
The explanation is still not sufficient but I've improved the things we discussed in the solution

**Comment by ahnlabb** [comment]
I will close this PR in favor of #17 .  (this PR is broken because I removed my old repository and forked anew)

**Pull request closed without merging**